test/test_classes.py

```diff
@@ -4,9 +4,8 @@
 
 """Unit testing of `pydot.classes`."""
 
-import pytest
-
 import pydot
+import pytest
 from pydot.classes import FrozenDict
```

Comment:
Wouldn't it make more sense to just upgrade the ruff version in the CI? :grin: 

Or are we back to, "Not compatible with Python 3.8"?

Reply:
You're 100% right. Maybe it should be a separate PR, but... too late.